.circleci/config.yml

```diff
@@ -727,6 +727,10 @@ jobs:
           name: Set USE_HERMES=1
           command: echo "export USE_HERMES=1" >> $BASH_ENV
 
+      - run:
+          name: Set BUILD_HERMES_SOURCE=1
+          command: echo "export BUILD_HERMES_SOURCE=1" >> $BASH_ENV
```

Comment:
This shouldn't be necessary.

Reply:
If we do not enable the build Hermes from the source, the newest changes with newly added methods for the profiler will not work. Are you sure?

Reply:
https://github.com/facebook/react-native/commit/fefa7b6ac8a1e0e33fa7a1070936c5c83c873c0a